src/pages/Proposal.vue

```diff
@@ -379,30 +371,44 @@ export default defineComponent({
 				};
 			}
 		},
-		async optInLsigToApp(lsig: LogicSigAccount) {
+		// funding lsig if the account has no algos
+		async checkLsigFund(lsig: LogicSigAccount) {
 			try {
-				if (typeof this.daoStore.dao_id === "undefined") {
-					this.error = APP_NOT_FOUND;
-					return;
-				}
-				if (typeof this.daoStore.govt_id === "undefined") {
-					this.error = TOKEN_NOT_FOUND;
-					return;
+				const response = await getAccountInfoByAddress(lsig.address());
+				let amount = null;
+				if (response) {
+					amount = response?.amount;
 				}
+				if (amount) return;
```

Comment:
There has to be min account that is needed by lsig.
@vuvoth Can you confirm this please?

Reply:
yes @vuvoth, can you tell me about it?

Reply:
Yes, I think so and this should be number algo we use for transfer when we create proprosal @Megha-Dev-19. Btw we should check the gov token when transfer. Did we do it?

Reply:
Like I talked to you about it, I will create a new PR for dao fund lsig

Reply:
And I will add the check for gov token transfer